### ISSUE TITLE
2.6.0 -> 2.6.1 regression

### ISSUE THREAD

**Issue by bbenezech** [joshwcomeau/react-flip-move#86]
2.6.0
![2 6 0](https://cloud.githubusercontent.com/assets/26794/19772032/33985c8e-9c65-11e6-9958-d606164f83bd.gif)
2.6.1
![2 6 1](https://cloud.githubusercontent.com/assets/26794/19772041/38993d98-9c65-11e6-8f23-a0723790926e.gif)

On the second screen, you can see that all elements but the last get removed at once instead of disappearing animated.

My setup is pretty basic:
```javascript
      <FlipMove className="card-list" staggerDelayBy="10">
        // list of <div className='card'></div>, block elements, no animation
     </FilpMove>
```

**Comment by joshwcomeau** [comment]
Huh, very strange. Sorry about that! I tested all the props but it's possible multiple simultaneous exits with staggers escaped my notice.

Hope it didn't cause any inconvenience. I'll take a stab at fixing it tonight. Feel free to lock your version number to 2.6.0 in the meantime.

**Comment by bbenezech** [comment]
@joshwcomeau Please, you owe me nothing, take all the time you need.
All it took was 2 minutes of copy pasting to get awesomely animated lists in my app.
A simple bug report is the least I can do to help you stabilize your refactoring. I'll stick to 2.6.0, no sweat.

I just tried removing the staggerDelayBy prop (sorry forgot about it) but no dice, the issue stays the same.

**Comment by joshwcomeau** [comment]
Hey @bbenezech,

Appreciate that :) I do feel bad though, since this was a patch release. I was debating making it a major release on the hunch that there _may_ be breaking changes, but decided against it since I couldn't find any. I'm worried it'll cause a lot of frustration when people can't figure out why stuff is suddenly broken after an NPM install!

I haven't had time to dig into it yet, but it should get done shortly.

**Comment by bbenezech** [comment]
Bugs and regressions are not desired, but expected. To prevent install-time regressions (and get deterministic builds*), your users can either:

1. lock to the last digit in `package.json`
2. use a shrinkwarp or yarn lock file

Shipping to prod without 1. or 2. is like playing soccer with a hand-grenade, it is their responsibility.

\* only for the first level of dependencies in case of 1.

**Comment by joshwcomeau** [comment]
Good points! Yarn's popularity should really help make deterministic dependency installation mainstream.

I've fixed the issue in v2.6.2. Turns out it was a very simple fix :)

**Comment by bbenezech** [comment]
Works perfectly thanks!
